sklearn/utils/multiclass.py

```diff
@@ -157,6 +157,24 @@ def is_multilabel(y):
         return len(labels) < 3 and (y.dtype.kind in 'biu' or  # bool, int, uint
                                     _is_integral_float(labels))
 
+def check_non_regression_targets(y):
```

Comment:
I think this function would better be named `check_classification_targets` instead. WDYT?